fgd/brush/trigger/trigger_momentum_timer_checkpoint.fgd

```diff
@@ -1,5 +1,9 @@
-@SolidClass base(ZoneTrigger)
-	appliesto(MOMENTUM)
-= trigger_momentum_timer_checkpoint : "Trigger used to denote progress in a linear map. trigger_momentum_timer_start is automatically checkpoint (zone number) 1!" 
+@SolidClass appliesto(MOMENTUM)
+= trigger_momentum_timer_checkpoint : "Zone used to denote progress in a linear map. trigger_momentum_timer_start is automatically checkpoint (zone number) 1!" 
 	[
+	targetname(target_source) : "Name" : : "The name of this zone."
```

Comment:
Is this needed?

Reply:
Checkpoints probably don't need a name, but stages and bonuses can have names based on the zone defs.